src/inmanta/deploy/tasks.py

```diff
@@ -217,7 +223,7 @@ async def execute(self, task_manager: "scheduler.TaskManager", agent: str, reaso
                 resource=self.resource,
                 attribute_hash=resource_details.attribute_hash,
                 status=state.ResourceStatus.UP_TO_DATE if success else None,
```

Comment:
I think we should get rid off `success` now and derive it from the deployment result.